### PR TITLE
Configurable WAL for PostgreSQL

### PR DESCRIPTION
**Pull request by dermusikman**
Our team would like to replicate WAL for backups/restores. This PR makes the following configurable in attributes for the postgresql.conf template:

wal_level
archive_mode
archive_command
archive_timeout

The default values are those for PostgreSQL.

### CODE REVIEW

**Comment by ryancragun** [comment]
:+1:

File: files/private-chef-cookbooks/private-chef/templates/default/postgresql.conf.erb [hunk 1]
```diff
@@ -180,10 +180,11 @@ checkpoint_warning = <%= node['private_chef']['postgresql']['checkpoint_warning'
 
 # - Archiving -
 
-#archive_mode = off		# allows archiving to be done
+
+archive_mode = <%= case node['private_chef']['postgresql']['archive_mode'] %>		# allows archiving to be done
```
Comment (irvingpop) [inline files/private-chef-cookbooks/private-chef/templates/default/postgresql.conf.erb]: What's the purpose of using `case` here?

Comment (dermusikman) [inline files/private-chef-cookbooks/private-chef/templates/default/postgresql.conf.erb]: Oops - leftovers from a previous iteration! Removed.

**Comment by irvingpop** [comment]
:+1:  looks good now

**Comment by sdelano** [comment]
I'm +1 on getting this merged in as well.

**Comment by stevendanna** [comment]
This has been merged here: https://github.com/chef/opscode-omnibus/commit/054c7ebef98c24ad15947064abf63ddc5da76d50  

Thank you for this contribution!

**Pull request closed without merging**